src/pages/students/[student_id]/goals/[goal_id]/benchmarks/[benchmark_id]/view.tsx

```diff
@@ -103,19 +107,32 @@ const ViewBenchmarkPage: NextPageWithBreadcrumbs = () => {
   const soloPoints: SoloPoint[] = [];
   const bulkPoints: BulkPoint[] = [];
 
-  const getDateFromPtNumber = (
-    pointNumber: number,
-    pointArray: DatePoint[]
-  ) => {
-    return new Date(pointArray[pointNumber].x).toDateString();
-  };
+  const allTrials: Array<{
```

Comment:
don't see this variable used anywhere

Reply:
ah oops, I'll unresolve this when you push the relevant commit (or GitHub will mark it "outdated")